src/status_im/utils/logging/core.cljs

```diff
@@ -41,6 +42,21 @@
     (get-js-logs)
     #(re-frame/dispatch [callback-handler %]))))
 
+(re-frame/reg-fx
+ :logs/set-level
+ (fn [log-level]
+   (log/set-level! log-level)))
+
+(fx/defn set-log-level
+  [{:keys [db]}  multiaccount]
+  (let [log-level (if-let [level (get multiaccount :log-level)]
+                    (if (clojure.string/blank? level) "ERROR" level)
+                    config/log-level-status-go)]
```

Comment:
I'm pretty sure there's a "nicer" way of doing this.

Reply:
@PascalPrecht could you please elaborate why if its nil we set log-level-status-go?

Reply:
@flexsurfer Yes sir! This was requested here: https://github.com/status-im/status-react/pull/10438#discussion_r415635013

Reply:
i knew that, thanks :)